frontend/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
is there any typescript being used in this project?

Reply:
There is now. :)